vendor/brave-ios/Ledger/Generated/NativeLedgerClientBridge.h

```diff
@@ -37,7 +37,7 @@
 - (void)onPanelPublisherInfo:(ledger::Result)result publisherInfo:(ledger::PublisherInfoPtr)publisher_info windowId:(uint64_t)windowId;
 - (void)onReconcileComplete:(ledger::Result)result viewingId:(const std::string &)viewing_id category:(ledger::REWARDS_CATEGORY)category probi:(const std::string &)probi;
 - (void)onRecoverWallet:(ledger::Result)result balance:(double)balance grants:(std::vector<ledger::GrantPtr>)grants;
-- (void)onRemoveRecurring:(const std::string &)publisher_key callback:(ledger::RecurringRemoveCallback)callback;
+- (void)onRemoveRecurring:(const std::string &)publisher_key callback:(ledger::RemoveRecurringTipCallback)callback;
```

Comment:
Minor nit: can we change the name of this method to match the new name in LedgerClient:

`onRemoveRecurring` → `removeRecurringTip`

Reply:
done